test/behaviour/reasoner/verification/SoundnessVerifier.java

```diff
@@ -107,7 +107,7 @@ void verifyQuery(TypeQLMatch inferenceQuery) {
 
     private boolean canExplanationBeVerified(Explanation explanation) {
         return iterate(explanation.conditionAnswer().concepts().values())
-                .filter(c -> c.asThing().isInferred() && !inferredConceptMapping.containsKey(c))
+                .filter(c -> c.isThing() && c.asThing().isInferred() && !inferredConceptMapping.containsKey(c))
```

Comment:
good catch - any idea why it wasn't done before? we just never had explanations with types in the answer?

Reply:
Unsure. Can undo change and run to remember.